src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,15 @@
+spring.application.name=TimeTracking
+spring.datasource.url=jdbc:mysql://172.16.4.17:3306/timetrackingdb?createDatabaseIfNotExist=true&useSSL=true
```

Comment:
I can't see the environement variables being used in any of the properties files

Reply:
I added an application-prod.properties file which will be used instead of application.properties when running the container

Reply:
Then what is the use of .env file ?